starport/interface/cli/starport/cmd/app.go

```diff
@@ -4,13 +4,25 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"time"
 
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing/object"
 	"github.com/gobuffalo/genny"
 	"github.com/spf13/cobra"
 	"github.com/tendermint/starport/starport/pkg/gomodulepath"
 	"github.com/tendermint/starport/starport/templates/app"
 )
 
+var (
+	commitMessage = "initialized w/ Starport"
+	devXAuthor    = &object.Signature{
+		Name:  "DevX Team @ Tendermint",
+		Email: "devx@tendermint.com",
```

Comment:
```suggestion
		Email: "hello@tendermint.com",
```

tbh, I don't want to monitor one more email address 😄